styx-scheduler-service/src/main/java/com/spotify/styx/state/StateManager.java

```diff
@@ -71,6 +81,20 @@ default void receiveIgnoreClosed(Event event) {
     }
   }
 
+  /**
+   * Like {@link #receive(Event)} but ignoring the {@link IsClosedException} exception.
+   *
+   * @param event The event to receive
+   * @param counter The state counter upon which the event must act upon
+   */
+  default void receiveIgnoreClosed(Event event, long counter) {
```

Comment:
Doesn't seem to be used.

Reply:
This will be gradually used when using explicit counters with StateManager